### ISSUE TITLE
How to trigger the uploading just when an input is changed

### ISSUE THREAD

**Issue by chiefGui** [blueimp/jQuery-File-Upload#3280]
Hello there.
### TL;DR

I am working on a Ruby on Rails application and I want to register a new user with its photo attached. The point is: I have a box only for the previsualization of the image and it is not sending when the form is finally submitted.
### The scenario itself

Think in a form. Inside it, there are some input fields such as name, email and so on; Also, there is a field to upload an image.

Next to this file field, there is a box displaying a preview of the uploaded image, what I'm using your plug-in to deal with. By then, two things should to happen: when the file field is `change`d, I need to trigger the `fieldupload` to send the desired image. In another hand, when the form itself is submitted, I also need to send the image, but then the proccess is no more related to the jQuery File Upload – they are separated things: previsualization and form upload.
#### But, what's the big deal?

This is my fragment of the jQuery File Upload's plug-in:

``` js
$('#photo').fileupload({
        dataType: 'json',
        url: '/coliseu/products/photo/upload/',
        done: function (e, data) {
          console.log(data);
            $.each(data.result.files, function (index, file) {
                $('<p/>').text(file.name).appendTo(document.body);
            });
        }
    });
```

If I don't remove it, the image is not sent with the form `submit`.

Thanks in advance.
